modules/hooks/activeEffects.js

```diff
@@ -1,22 +1,22 @@
 
 export default function () {
 
-    Hooks.on("createActiveEffect", (effect, options, id) => {_runUpdateEffects(effect, "create", options, id)})
-    Hooks.on("updateActiveEffect", (effect, options, id) => {_runUpdateEffects(effect, "update", options, id)})
-    Hooks.on("deleteActiveEffect", (effect, options, id) => {
+    Hooks.on("createActiveEffect", async (effect, options, id) => {await _runUpdateEffects(effect, "create", options, id)})
+    Hooks.on("updateActiveEffect", async (effect, options, id) => {await _runUpdateEffects(effect, "update", options, id)})
+    Hooks.on("deleteActiveEffect", async (effect, options, id) => {
         if (effect.parent.documentName == "Actor")
         {
             let items = effect.parent.items.filter(i => i.getFlag("wfrp4e", "fromEffect") == effect.id);
             if (items.length)
             {
                 ui.notifications.notify(game.i18n.format("EFFECT.DeletingItems", {items : items.map(i => i.name).join(", ")}))
-                effect.parent.deleteEmbeddedDocuments("Item", items.map(i => i.id));
+                await effect.parent.deleteEmbeddedDocuments("Item", items.map(i => i.id));
             }
         }
-        _runUpdateEffects(effect, "delete", options, id)
+        await _runUpdateEffects(effect, "delete", options, id)
```

Comment:
These don't seem necessary

Reply:
wanted to explicilty show, that those effects are awaitable, i can remove.